lib/rijksmus-api/version.rb

```diff
@@ -1,5 +1,5 @@
 module Rijksmus
   module API
-    VERSION = "0.1.0"
+    VERSION = '0.1.0'.freeze
```

Comment:
Why use .freeze here?

Reply:
"rubocop -a" is doing his thing" :).
And he teach us the new syntaxes and clean/safer better ways to do many things.
[Here is the explanation.](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/MutableConstant)